README.md

```diff
@@ -398,6 +398,9 @@ whereNotBetween($field, $value) | whereNotBetween('price', [100, 200]) | Checks
 whereExists($field) | whereExists('unemployed') | Checks if a value is defined.
 whereNotExists($field) | whereNotExists('unemployed') | Checks if a value isn't defined.  
 whereRegexp($field, $value, $flags = 'ALL') | whereRegexp('name.raw', 'A.+') | Filters records according to a given regular expression. [Here](https://www.elastic.co/guide/en/elasticsearch/reference/5.2/query-dsl-regexp-query.html#regexp-syntax) you can find more about syntax.
+whereGeoDistance($field, $value, $distance) | whereGeoDistance('location', [-70, 40], 1000) | Filters records according to given point and distance from it. [Here](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-geo-distance-query.html) you can find more about syntax.
```

Comment:
Are you sure that you can specify `1000` like a number without a unit? I'd put here `1000km`.

Reply:
default in meters

Reply:
Still I'd specified unit for an example, it makes it more clear.